### ISSUE TITLE
Stop!

### ISSUE THREAD

**Issue by ghost** [ModdingTwins/PocketGameCore#3]
Why You use my LobbyPlugin?and Change author? I will Strike it.

**Comment by uvsq22401387** [comment]
That is true, he wont stop editing the plugins of the others that why i opened issues and he 
closed them!
